scripts/tests

```diff
@@ -78,9 +84,70 @@ def count(bstr, match):
     return str(bstr, encoding="utf8").count(match)
 
 
+# constants
+
+
+class RNG(Enum):
+    HAL = 1
+    NOTRAND = 2
+    NISTKAT = 3
+
+    def __str__(self):
+        return self.name
+
+
+class TEST_TYPE(Enum):
+    TEST = 1  # functional test
+    SPEED = 2
+    STACK = 3
+    NISTKAT = 4
+
+    def __str__(self):
+        return self.name.lower()
+
+
+class PLATFORM(Enum):
+    STM32F4DISCOVERY = 1
+    MPS2_AN386 = 2
+    STM32F767ZI = 3
```

Comment:
I think the naming is a little unfortunate. 
` STM32F767ZI` is the name of the CPU, but the name of the platform is actually `NUCLEO-F767ZI`.
It's probably a good idea to use `NUCLEO-F767ZI` (also to be in line with pqm4).